app/Actions/Diagnostics/Pipes/Checks/AppUrlMatchCheck.php

```diff
@@ -50,7 +61,7 @@ public function handle(array &$data, \Closure $next): array
 			$data[] = 'Error: LYCHEE_UPLOADS_URL is set and empty. This will prevent images to be displayed. Remove the line from your .env';
 		}
 
-		if (($config_url . '/uploads/') === $config_url_imgage && !$this->checkUrlMatchCurrentHost()) {
+		if (($config_url . ($dir_url !== '' ? $dir_url . '/' : '') . '/uploads/') === $config_url_imgage && !$this->checkUrlMatchCurrentHost()) {
```

Comment:
```suggestion
		if (($config_url . $dir_url . '/uploads/') === $config_url_imgage && !$this->checkUrlMatchCurrentHost()) {
```

I agree with @d7415 above but once the extra `'/'` is eliminated, it can simply be like this?

Reply:
I also just noticed the "imgage", which I assume is a typo, but is the correct variable name 🤷‍♂️